docs/samples/Microsoft.ML.Samples/Dynamic/DataOperations/BootstrapSample.cs

```diff
@@ -12,24 +11,17 @@ public static void Example()
             // as a catalog of available operations and as the source of randomness.
             var mlContext = new MLContext();
 
-            // Get a small dataset as an IEnumerable and them read it as ML.NET's data type.
-            IEnumerable<Microsoft.ML.SamplesUtils.DatasetUtils.BinaryLabelFloatFeatureVectorFloatWeightSample> enumerableOfData = Microsoft.ML.SamplesUtils.DatasetUtils.GenerateBinaryLabelFloatFeatureVectorFloatWeightSamples(5);
-            var data = mlContext.Data.LoadFromEnumerable(enumerableOfData);
+            // Get a small dataset as an IEnumerable.
+            var rawData = new[] {
+                new DataPoint() { Label = true, Feature = 1.017325f},
+                new DataPoint() { Label = false, Feature = 0.6326591f},
+                new DataPoint() { Label = false, Feature = 0.0326252f},
+                new DataPoint() { Label = false, Feature = 0.8426974f},
+                new DataPoint() { Label = true, Feature = 0.9947656f},
+                new DataPoint() { Label = true, Feature=1.017325f},
```

Comment:
>Feature=1 [](start = 48, length = 9)

nit: add extra space.

Good job on picking up this SampleUtils reference!

Reply:
[https://media0.giphy.com/media/YYfEjWVqZ6NDG/source.gif](https://media0.giphy.com/media/YYfEjWVqZ6NDG/source.gif)

---
In reply to: [277856475](https://github.com/dotnet/machinelearning/pull/3539#discussion_r277856475) [](ancestors = 277856475)